clang/lib/Sema/Sema.cpp

```diff
@@ -967,6 +967,10 @@ static void checkUndefinedButUsed(Sema &S) {
                            Func->getIdentifier()->isMangledOpenMPVariantName();
         }
       }
+      // Do not warn on undefined internal functions in HLSL, they will get
```

Comment:
dxc do get warning 
Here's an example
https://godbolt.org/z/qPbvEzWzM

Will this change in clang?

Reply:
Good catch! I will bring the warning back and modify tests that hit it.